### PR TITLE
refactor(cce): refactor cce resource codes

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
Refactor cce resource codes:
- adjust function names.
- adjust log and error message.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. Refactor CCE resource codes.
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
NONE
```

### CODE REVIEW

File: huaweicloud/services/cce/resource_huaweicloud_cce_node_attach.go [hunk 1]
```diff
@@ -377,12 +386,12 @@ func resourceCCENodeAttachV3Create(ctx context.Context, d *schema.ResourceData,
 	}
 	addOpts.NodeList[0].Spec.Login = loginSpec
 
-	s, err := nodes.Add(nodeClient, clusterID, addOpts).ExtractAddNode()
+	s, err := nodes.Add(cceClient, clusterID, addOpts).ExtractAddNode()
 	if err != nil {
-		return fmtp.DiagErrorf("Error adding HuaweiCloud Node: %s", err)
+		return diag.Errorf("error adding node to : %s", err)
```
Comment (ShiChangkuo) [inline huaweicloud/services/cce/resource_huaweicloud_cce_node_attach.go]: adding node to CCE cluster?

File: huaweicloud/services/cce/resource_huaweicloud_cce_addon.go [hunk 2]
```diff
@@ -379,23 +370,22 @@ func waitForAddonDelete(cceClient *golangsdk.ServiceClient, id, clusterID string
 		if r.Status.Status == "Deleting" {
 			return r, "Deleting", nil
 		}
-		log.Printf("[DEBUG] CCE Addon %s still available.\n", id)
+		log.Printf("[DEBUG] CCE add-on %s still available.", id)
```
Comment (ShiChangkuo) [inline huaweicloud/services/cce/resource_huaweicloud_cce_addon.go]:
```suggestion
		log.Printf("[DEBUG] CCE add-on %s is still available.", id)
```

File: huaweicloud/services/cce/resource_huaweicloud_cce_node_attach.go (hunk 1 above)
Comment (Lance52259) [reply huaweicloud/services/cce/resource_huaweicloud_cce_node_attach.go]: fixed and bring this file change to another PR.

File: huaweicloud/services/cce/resource_huaweicloud_cce_addon.go (hunk 2 above)
Comment (Lance52259) [reply huaweicloud/services/cce/resource_huaweicloud_cce_addon.go]: fixed and bring this file change to another PR.

File: huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go [hunk 3]
```diff
@@ -846,37 +851,35 @@ func waitForCceNodePoolActive(cceClient *golangsdk.ServiceClient, clusterId, nod
...
+		log.Printf("[DEBUG] CCE node pool (%s) is still available.", nodePoolId)
 		return r, r.Status.Phase, nil
 	}
 }
 
-func resourceCCENodePoolV3Import(_ context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
-	parts := strings.SplitN(d.Id(), "/", 2)
+func resourceNodePoolImport(_ context.Context, d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData, error) {
+	importId := d.Id()
+	parts := strings.SplitN(importId, "/", 2)
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go]:
```suggestion
	parts := strings.Split(importId, "/")
```

File: huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go [hunk 4]
```diff
@@ -645,30 +657,21 @@ func resourceCCENodePoolRead(_ context.Context, d *schema.ResourceData, meta int
...
+		rootVolume["kms_key_id"] = s.Spec.NodeTemplate.RootVolume.Metadata.SystemCmkid
 	}
-	mErr = multierror.Append(mErr, d.Set("root_volume", rootVolume))
-
-	tagmap := utils.TagsToMap(s.Spec.NodeTemplate.UserTags)
-	mErr = multierror.Append(mErr,
-		d.Set("tags", tagmap),
-		d.Set("status", s.Status.Phase),
-	)
+	mErr = multierror.Append(mErr, d.Set("root_volume", []interface{}{rootVolume}))
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go]: you can use flattenResourceNodeRootVolume

File: huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go [hunk 5]
```diff
@@ -645,30 +657,21 @@ func resourceCCENodePoolRead(_ context.Context, d *schema.ResourceData, meta int
 	}
 	mErr = multierror.Append(mErr, d.Set("data_volumes", volumes))
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go]: you can use flattenResourceNodeDataVolume

File: huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go [hunk 6]
```diff
@@ -550,51 +560,51 @@ func resourceCCENodePoolCreate(ctx context.Context, d *schema.ResourceData, meta
 	}
 	createOpts.Spec.NodeTemplate.Login = loginSpec
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go:527]: you can use buildResourceNodeLoginSpec

Comment (Lance52259) [reply huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go:527]: done.

File: huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go (hunk 5 above)
Comment (Lance52259) [reply huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go]: done.

File: huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go (hunk 4 above)
Comment (Lance52259) [reply huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go]: done.

File: huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go (hunk 3 above)
Comment (Lance52259) [reply huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go]: done.

**Pull request merged**